samples/cpp/stitching_detailed.cpp

```diff
@@ -119,8 +119,10 @@ static void printUsage()
         "      Blending strength from [0,100] range. The default is 5.\n"
         "  --output <result_img>\n"
         "      The default is 'result.jpg'.\n"
-        "  --timelapse (as_is|crop) (range_width)\n"
-        "      Output warped images separately as frames of a time lapse movie, with 'fixed_' prepended to input file names.\n";
+        "  --timelapse (as_is|crop) \n"
+        "      Output warped images separately as frames of a time lapse movie, with 'fixed_' prepended to input file names.\n"
+        "  --rangewidth <int>\n"
+        "      uses range_width to limit number of images to match with.\n";
 }
```

Comment:
Please look at this comment about OpenCV CommandLineParser: 
https://github.com/Itseez/opencv/pull/4110#discussion_r42940259